src/sentry/tasks/auto_ongoing_issues.py

```diff
@@ -93,38 +99,28 @@ def schedule_auto_transition_new() -> None:
 @retry(on=(OperationalError,))
 @skip_if_queue_has_items
 def auto_transition_issues_new_to_ongoing(
-    project_id: int,
+    project_ids: List[int],
     first_seen_lte: int,
-    first_seen_gte: Optional[int] = None,
-    chunk_size: int = 1000,
     **kwargs,
 ) -> None:
```

Comment:
It's probably a good idea to keep `project_id` around with a default value, same for `project_ids`. This just helps us avoid any problems while deploying, where some workers might expect `project_ids` to be passed, and other `project_id`. After we've deployed then you can clean up the signature in a separate pr.